src/VisualStudio/Setup/VisualStudioSetup.csproj

```diff
@@ -25,6 +25,10 @@
     <DefineConstants>$(DefineConstants);OFFICIAL_BUILD</DefineConstants>
   </PropertyGroup>
   <ItemGroup Label="Project References">
+    <ProjectReference Include="..\..\Compilers\Core\MSBuildTask\MSBuildTask.csproj">
+      <Project>{d874349c-8bb3-4bdc-8535-2d52ccca1198}</Project>
```

Comment:
Does this result in the task ending up in the VSIX? Do you need <ReferenceOutputAssembly> here?